packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -129,6 +203,71 @@ class ScrollBehavior {
   String toString() => objectRuntimeType(this, 'ScrollBehavior');
 }
 
+class _WrappedScrollBehavior implements ScrollBehavior {
```

Comment:
@goderbauer this does not appear to work in the way you thought it might. When `_WrappedScrollBehavior` calls `delegate.buildViewportDecoration`, it does not in turn call `_WrappedScrollBehavior.buildScrollbar`, it calls `ScrollBehavior.buildScrollbar`. It ignores the implemented class once you call on the delegate. Maybe I misunderstood what you preferred?

Reply:
Right, I think _WrappedScrollBehavior will have to re-implement buildViewportDecoration and call those methods on itself.

Reply:
Won't that create an issue though if the `ScrollBehavior` being copied to `_WrappedScrollBehavior` has a different implementation of `buildViewportDecoration`? 
This is why I was trying to keep everything contained in `buildViewportDecoration` previously.